taglib/src/main/java/com/reevoo/taglib/AbstractReevooMarkClientTag.java

```diff
@@ -119,7 +107,20 @@ protected void setClient(ReevooMarkClient client) {
             // with the parameter "reviews" instead of the parameter "per_page".
             queryStringParams.put("reviews",this.numberOfReviews);
         }
+        if (this.dynamicAttrs != null) {
+          for (Map.Entry<String,String> entry : this.dynamicAttrs.entrySet()) {
+            queryStringParams.put(this.to_camel_case(entry.getKey()), entry.getValue());
+          };
+        }
         return queryStringParams;
     }
 
+
+    /**
+     * Transforms attribute name from camel case to snake case.
+     */
+    protected String to_camel_case(String attributeName) {
```

Comment:
This method name should be in camel case to respect the Java naming conventions.